api/api/src/main/scala/uk/ac/wellcome/platform/api/models/WorkQuery.scala

```diff
@@ -32,7 +33,8 @@ object WorkQuery {
         queryString,
         fields = defaultBoostedFields,
         lenient = Some(true),
-        minimumShouldMatch = Some(defaultMSM)
+        minimumShouldMatch = Some(defaultMSM),
+        flags = Seq(SimpleQueryStringFlag.PHRASE)
```

Comment:
Maybe a comment here explaining why we set this flag, and the sort of queries that break if we don’t?